test/support/test_config.h

```diff
@@ -52,11 +52,7 @@
 // GCC10 produces wrong answer calling exclusive_scan using vectorized polices
 #define TEST_GCC10_EXCLUSIVE_SCAN_BROKEN (_GLIBCXX_RELEASE == 10)
 // GCC7 std::get doesn't return const rvalue reference from const rvalue reference of tuple
-#if defined(_GLIBCXX_RELEASE)
-#   define _PSTL_TEST_GCC7_RVALUE_TUPLE_GET_BROKEN (_GLIBCXX_RELEASE < 8)
-#else
-#   define _PSTL_TEST_GCC7_RVALUE_TUPLE_GET_BROKEN 0
-#endif
+#define _PSTL_TEST_GCC7_RVALUE_TUPLE_GET_BROKEN (_GLIBCXX_RELEASE > 0 && _GLIBCXX_RELEASE < 8)

```

Comment:
This part `_GLIBCXX_RELEASE > 0` is mandatory required for the case when `_GLIBCXX_RELEASE` is undefined - for example on Windows compilers and etc.